### PR TITLE
Add full class names as a comment for element and modifier classes in BEM nesting

### PR DESCRIPTION
**Pull request by jaakritso**
Full class names as comments are especially useful if you're searching for particular classname from the whole directory.

So the searching results are similar to eCSStractor without nesting option.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ With BEM nesting:
 
 ![ecss_bem](https://cloud.githubusercontent.com/assets/654597/5896785/60708c5c-a54c-11e4-963f-9e00ede168c3.gif)
 
+With BEM nesting and class names as comments:
+![ecss_bem_comments](https://raw.githubusercontent.com/jaakritso/ecsstractor/master/eCSStractor-comments.mov.gif)
```
Comment (hudochenkov) [inline README.md]: Replace image code with `ecss_bem_comments`. This way it will link to the image in the correct repository.

**Review by hudochenkov — CHANGES_REQUESTED** [review]
Thank you for your addition! And thank you for your patience waiting for a review.

Could you, please, use CSS comments (`/* comment */`) instead of SCSS comment (`// comment`). Normal comments could be used in many syntaxes (including SCSS), while SCSS comments are very limited.

**Comment by jaakritso** [comment]
The need for that kind of comments is necessary only for the nested markup, as the class names are chopped into pieces. In regular CSS it's not possible to write nested CSS anyways ( !? ), so the regular comments are not needed, in my opinion.

But I'll look into it, I'll try to do it so that the default class comments are CSS style, but from the options it's possible to set it as SCSS comments as well.

**Comment by hudochenkov** [comment]
It also possible to use PostCSS with [postcss-nested](https://github.com/postcss/postcss-nested) plugin. Default PostCSS parser doesn't understand `//` comments.

**Review by jaakritso — COMMENTED** [review]
Added the default comment style to regular CSS comments and kept the SCSS style comments as a option that is user configurable.

**Review by hudochenkov — APPROVED** [review]

**Comment by hudochenkov** [comment]
Thanks!

**Comment by hudochenkov** [comment]
New version `1.8.0` will be available within two hours.

**Pull request merged**